WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginGoogleFragment.java

```diff
@@ -1,299 +1,124 @@
                             showErrorDialog(getString(R.string.login_error_generic));
                         }
                     } else {
                         AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_SOCIAL_BUTTON_FAILURE);
                         switch (signInResult.getStatus().getStatusCode()) {
                             // Internal error.
                             case GoogleSignInStatusCodes.INTERNAL_ERROR:
-                                AppLog.e(T.NUX, "Google Sign-in Failed: internal error.");
+                                AppLog.e(T.NUX, "Google Login Failed: internal error.");
                                 showErrorDialog(getString(R.string.login_error_generic));
                                 break;
                             // Attempted to connect with an invalid account name specified.
                             case GoogleSignInStatusCodes.INVALID_ACCOUNT:
-                                AppLog.e(T.NUX, "Google Sign-in Failed: invalid account name.");
+                                AppLog.e(T.NUX, "Google Login Failed: invalid account name.");
                                 showErrorDialog(getString(R.string.login_error_generic)
                                         + getString(R.string.login_error_suffix));
                                 break;
                             // Network error.
                             case GoogleSignInStatusCodes.NETWORK_ERROR:
-                                AppLog.e(T.NUX, "Google Sign-in Failed: network error.");
+                                AppLog.e(T.NUX, "Google Login Failed: network error.");
                                 showErrorDialog(getString(R.string.error_generic_network));
                                 break;
                             // Cancelled by the user.
                             case GoogleSignInStatusCodes.SIGN_IN_CANCELLED:
-                                AppLog.e(T.NUX, "Google Sign-in Failed: cancelled by user.");
+                                AppLog.e(T.NUX, "Google Login Failed: cancelled by user.");
                                 break;
                             // Attempt didn't succeed with the current account.
                             case GoogleSignInStatusCodes.SIGN_IN_FAILED:
-                                AppLog.e(T.NUX, "Google Sign-in Failed: current account failed.");
+                                AppLog.e(T.NUX, "Google Login Failed: current account failed.");
                                 showErrorDialog(getString(R.string.login_error_generic));
                                 break;
                             // Attempted to connect, but the user is not signed in.
                             case GoogleSignInStatusCodes.SIGN_IN_REQUIRED:
-                                AppLog.e(T.NUX, "Google Sign-in Failed: user is not signed in.");
+                                AppLog.e(T.NUX, "Google Login Failed: user is not signed in.");
```

Comment:
[The docs](https://developers.google.com/android/reference/com/google/android/gms/common/ConnectionResult.html#SIGN_IN_REQUIRED) are confusing, but if I'm reading correctly then when `SIGN_IN_REQUIRED` is returned we may be able to continue with sign in (?)

Reply:
Yeah, the terminology is a little confusing.  We've tried to differentiate by using _login_ when referring to an existing account and _signup_ when talking about creating a new account.  The term sign-in used by the API and Google documentation refers to both login and signup depending on the situation, but the `SIGN_IN_REQUIRED` in this case means login.